tests/end_to_end_integration/argo.yaml

```diff
@@ -376,6 +376,8 @@ spec:
             value: "{{workflow.parameters.nudge-to-fine-config}}"
           - name: segment-count
             value: "{{workflow.parameters.nudge-to-fine-segment-count}}"
+          - name: image-tags
```

Comment:
I feel some reluctance to add a workflow parameter for this, since it possible that this parameter could be inconsistent with the actual tag. `COMMIT_SHA` is an environmental variable in both the fv3net and prognostic run images. Could you use that instead?

There are also mechanisms to get the container name/tag using k8s. https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/.

In the interest of moving forward with this PR, it might make more sense to drop this git-sha functionality and discuss it at a later point. It's a somewhat independent feature.

Reply:
Will separate that out